server/routes/v1/version.spec.ts

```diff
@@ -27,6 +51,65 @@ describe('test version routes', () => {
     expect(res.body.version).toBe('1')
   })
 
+  test('that when we delete a version, we send an email to the relvant deployment owners', async () => {
+    await ModelModel.create(testModel)
+    const spy: any = jest.spyOn(approvalService, 'deleteApprovalsByVersion')
+    spy.mockReturnValue({ irrelevant: 'content' })
+    const updateImplementation = (deploymentService.findDeploymentsByModel as jest.Mock).mockImplementationOnce(() => [
```

Comment:
Doesn't really matter but you could instead use `mockReturnValueOnce(['deployment'])`. `mockImplementation` is more useful if you want to change the functionality of something in a test (which is pretty uncommon)